### PR TITLE
Live Event Schedule

### LINKED ISSUE #84: Event schedule - DESKTOP + MOBILE
- [ ] Create a component EventScheduleSection inside of live site and render it on the index page of the live site
- [ ] The component should have everything except for the white boxes (that will be a separate ticket) 
- [ ] Make sure your component manages the state of which day is selected
Screenshot 2022-11-30 at 12 23 46 AM

### PR DESCRIPTION
**Pull request by mike10911**
Added Live Event Schedule without events

Fixes #84 .

Changelist:

- Added event tab component
- put mock data in `EventScheduleMockData.tsx`
- added `EventScheduleTabProps` and `StyledTabTitleWrapperProps` to live site `types.ts`

Notes:

- Not sure if the button is supposed to change shape

Tested:

- localhost on chrome and safari

Screenshots & Screencasts:

https://user-images.githubusercontent.com/103906123/207242017-b93bf97d-ff5e-488e-952b-c53f572fc72b.mov

### CODE REVIEW

File: live-site/components/event-schedule-section/EventScheduleSection.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleSection.styles.tsx]: Remove this if you're not adding any styles - just use the PrimaryButton as is

File: live-site/components/event-schedule-section/EventScheduleTab.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleTab.styles.tsx:13]: Why is there margin-right and top 0 if margin is 5em?

File: live-site/components/event-schedule-section/EventScheduleTab.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleTab.styles.tsx]: Remove this if there is no styling

File: live-site/components/event-schedule-section/EventScheduleSection.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleSection.tsx:34]: Not a blocker for getting this ticket merged in but would you mind asking around to find out who is supposed to create this link?

Comment (jzhzhang66) [reply live-site/components/event-schedule-section/EventScheduleSection.tsx:34]: and maybe bump them on it if it's not on their radar yet

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
MIKE THIS LOOKS AMAZINGGGG U KILLLED IT OMGGGG. So proud of how much you've grown as a front end dev WOOHOOOOOO!!!! 

I just left a few minor comments and I'll be ready to approve after!

File: live-site/components/event-schedule-section/EventScheduleSection.tsx
Comment (mike10911) [reply live-site/components/event-schedule-section/EventScheduleSection.tsx:34]: @jzhzhang66 Okay! I asked David and Sreya!

File: live-site/components/event-schedule-section/EventScheduleSection.styles.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleSection.styles.tsx]: This styled component is still here - remove this and all references to it since you didn't add any styles!

File: live-site/components/event-schedule-section/EventScheduleSection.tsx
Comment (jzhzhang66) [inline live-site/components/event-schedule-section/EventScheduleSection.tsx]: I'm pretty sure it's spelled "Calendar"

**Review by jzhzhang66 — APPROVED** [review]
LGTM - plz just fix the 2 comments I left before merging!

**Review by jhylisa666 — APPROVED** [review]
LGTMMM

**Pull request merged**
